101-data-factory-chained-copy-activities/azuredeploy.json

```diff
@@ -0,0 +1,370 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "dataLakeStoreUri": {
+        "type": "string",
+        "metadata": {
+          "description": "URI of Azure Data Lake store"
+        }
+      },
+      "dataLakeStoreServicePrincipalID": {
+        "type": "string",
+        "metadata": {
+          "description": "ID of Azure Service Principal used for accessing Data Lake"
+        }
+      },
+      "dataLakeStoreServicePrincipalKey": {
+        "type": "securestring",
+        "metadata": {
+          "description": "Key for Azure Service Principal used for accessing Data Lake"
+        }
+      },
+      "azureSQLConnectionString": {
+        "type": "securestring",
+        "metadata": {
+          "description": "Connection string for Azure SQL Database"
+        }
+      }
+    },
+    "variables": {
+      "dataFactoryName": "[concat('CopyFromSQLToADFToAzureSQL', uniqueString(resourceGroup().id))]",
```

Comment:
does this need to be globally unique or just unique within the RG?  i.e. can we remove uniqueString?

Reply:
Data Factory names are unique across Azure so the uniqueString is required: https://docs.microsoft.com/en-us/azure/data-factory/data-factory-naming-rules